docs/resources/service-providers.md

```diff
@@ -56,25 +168,26 @@ gaiad config node <host>:<port
 // example: gaiad config node https://77.87.106.33:26657
 ```
 
-If you run your own full-node, just use `tcp://localhost:26657` as the address. 
+If you run your own Full Node locally, use `tcp://localhost:26657` as the address. 
 
-Then, let us set the default value of the `--trust-node` flag:
+Set the default value of the `--trust-node` flag:
 
 ```bash
 gaiad config trust-node false
 
-// Set to true if you run a light-client node, false otherwise
+// Set to true if you run a light-client node
```

Comment:
```suggestion
// Set to true if you run a light client node
```

Reply:
do we have a target page to link to for light clients?

Reply:
I do not think we have this yet